doc/sphinx/language/core/records.rst

```diff
@@ -6,8 +6,8 @@ Record types
 The :cmd:`Record` construction is a :term:`command` allowing the definition of
 records as is done in many programming languages. Its syntax is described in the
 grammar below. In fact, the :cmd:`Record` :term:`command` is more general than
-the usual record types, since it allows also for “manifest”expressions. In this
-sense, the :cmd:`Record` construction allows defining“signatures”.
+the usual record types, since it allows also for “manifest” expressions. In this
+sense, the :cmd:`Record` construction allows defining “signatures”.
```

Comment:
FWIW, these really shouldn't be in quotes and there should be definitions somewhere.  And maybe these should be listed in the glossary.